Testscripts/Linux/perf_apache.sh

```diff
@@ -0,0 +1,255 @@
+#!/bin/bash
+
+#######################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+#######################################################################
+
+#######################################################################
+#
+# perf_apache.sh
+# Description:
+#    Install apache and run apache performance tests.
+#    This script needs to be run on client VM.
+#
+# Supported Distros:
+#    Ubuntu/Suse/Centos/RedHat/Debian
+#######################################################################
+
+CONSTANTS_FILE="./constants.sh"
+UTIL_FILE="./utils.sh"
+
+. ${CONSTANTS_FILE} || {
+	errMsg="Error: missing ${CONSTANTS_FILE} file"
+	LogErr "${errMsg}"
+	SetTestStateAborted
+	exit 10
```

Comment:
What is this magic number ? Any specific intention? Exit with 0 by setting correct state else framework may retry.

Reply:
Fixed.